src/main/java/org/osiam/resources/controller/GroupController.java

```diff
@@ -117,4 +131,13 @@ public void delete(@PathVariable final String id) {
         headers.setLocation(location);
         return new ResponseEntity<>(group, headers, status);
     }
+
+    private Set<String> extractAttributes(String attributesParameter) {
+        List<String> strings = Lists.newArrayList("schemas");
```

Comment:
The reason is that I wasn't secure enough in my streams api usage. Using the Splitter makes the whole discussion moot. However, I just consulted the scim RFC and we have a bug here: not "schemas" but "id" is supposed to be always returned. I'll file a bug.